pkg/render/compliance.go

```diff
@@ -713,31 +722,34 @@ func (c *complianceComponent) complianceServerPodSecurityPolicy() *policyv1beta1
 	return psp
 }
 
-func complianceVolumeMounts(managerSecret *corev1.Secret) []corev1.VolumeMount {
+func (c *complianceComponent) complianceVolumeMounts() []corev1.VolumeMount {
 	var mounts = []corev1.VolumeMount{{
 		Name:      "cert",
 		MountPath: "/code/apiserver.local.config/certificates",
 		ReadOnly:  true,
 	}}
 
-	if managerSecret != nil {
+	if c.managerInternalTLSSecret != nil {
 		mounts = append(mounts, corev1.VolumeMount{
 			Name:      ManagerInternalTLSSecretName,
 			MountPath: "/manager-tls",
 			ReadOnly:  true,
 		})
 	}
 
+	if c.dexCfg != nil {
+		mounts = append(mounts, c.dexCfg.RequiredVolumeMounts()...)
+	}
+
 	return mounts
 }
 
-func complianceVolumes(defaultMode int32, managerSecret *corev1.Secret) []corev1.Volume {
+func (c *complianceComponent) complianceVolumes() []corev1.Volume {
 	var volumes = []corev1.Volume{{
 		Name: "cert",
 		VolumeSource: corev1.VolumeSource{
 			Secret: &corev1.SecretVolumeSource{
-				DefaultMode: &defaultMode,
```

Comment:
Did you mean to remove this?

Reply:
yes, this is the default value that k8s otherwise puts here.